archivers/libzzip/Portfile

```
@@ -64,14 +57,43 @@ post-destroot {
 }
 
 variant sdl description {Enable SDL support} {
-    depends_lib-append      port:libsdl
+    depends_lib-append      port:libsdl2
 
-    configure.args-append   --enable-sdl
+    configure.args-append   -DZZIPSDL=ON
 
     post-destroot {
         xinstall -m 0644 ${worksrcpath}/docs/README.SDL ${destroot}${docdir}
     }
 }
 
-test.run        yes
-test.target     check
+variant python35 conflicts python36 python37 python38 python39 description {Build using Python 3.5} {
+    depends_lib-append \
+        port:python35
+    configure.args-append -DPYTHON_EXECUTABLE="${prefix}/bin/python3.5"
+}
+
+variant python36 conflicts python35 python37 python38 python39 description {Build using Python 3.6} {
+    depends_lib-append \
+        port:python35
+    configure.args-append -DPYTHON_EXECUTABLE="${prefix}/bin/python3.6"
+}
+
+variant python37 conflicts python35 python36 python38 python39 description {Build using Python 3.7} {
+    depends_lib-append \
+        port:python37
+    configure.args-append -DPYTHON_EXECUTABLE="${prefix}/bin/python3.7"
+}
+
+variant python38 conflicts python35 python36 python37 python39 description {Build using Python 3.8} {
+    depends_lib-append \
+        port:python38
+    configure.args-append -DPYTHON_EXECUTABLE="${prefix}/bin/python3.8"
+}
+
+variant python39 conflicts python35 python36 python37 python38 description {Build using Python 3.9} {
+    depends_lib-append \
+        port:python39
+    configure.args-append -DPYTHON_EXECUTABLE="${prefix}/bin/python3.9"
+}
+
+default_variants +python39
```

This should only be the default it none of the other conflicting python variants have been selected. (MacPorts use of the term "default" here is a bit nonstandard.) See https://trac.macports.org/wiki/PortfileRecipes#default_variants

---

I'm unsure as to how handle this then.
Originally the library uses deprecated `FindPythonInterp`.

Maybe a file dependency on `python3` instead and do away with the variants altogether?

---

```suggestion
if {![variant_isset python35] && ![variant_isset python36] && ![variant_isset python37] && ![variant_isset python38] && ![variant_isset python39]} {
    default_variants +python39
}
```
Or just depend unconditionally on python39 and remove the variants.